pipeline_config/pipeline-conditional-processing.yaml

```diff
@@ -0,0 +1,110 @@
+spec: #defined at publish pipeline
```

Comment:
Can we make this version cleaner by removing the commented code and inserting a brief overview of the pipeline? For example, what it does in plain language.

We can store these lines of code in a separate file probably. Your call.

Reply:
These are the command line calls that are equivalent to the yaml file, that's why I put them here as a reference. But you are right, I can remove them out to another place, e.g. notes.